lib/salus/scanners/brakeman.rb

```diff
@@ -133,6 +133,9 @@ def user_supplied_ignore_expirations?
         return true if item.key?('expiration')
       end
       false
+    rescue TypeError, Errno::ENOENT => e
+      report_error(e.message)
+      bugsnag_notify(e)
```

Comment:
Please delete `bugsnag_notify`, because `report_error` already calls `bugsnag_notify`.
Ref: https://github.com/coinbase/salus/blob/e66332c8479dd4b220cc22af9f588abc8848dfc0/lib/salus/scanners/base.rb#L188-L194